pkg/cpi/mockingUtils.go

```diff
@@ -438,6 +438,8 @@ func GetCPIFunctionNameByURLCheck(url, method, testType string) string {
 		return GetFunctionNameByTestTypeAndMethod(method, testType)
 	case "https://demo/api/v1/IntegrationRuntimeArtifacts('flow1')":
 		return "GetIntegrationArtifactDeployStatus"
```

Comment:
Now that you've changed the test, I think you can remove this outdated case

Reply:
removed